.gitignore

```diff
@@ -217,3 +217,9 @@ tools/build/releases/*
 # Docker configuration
 install.lock
 docker-compose.override.yml
+
+# DotEnv files
+/.env
```

Comment:
.env should not be ignore

Reply:
Hmm why the .env.dist should be indeed versionized as an example but not. the .env It's a custom setting file for each environment so it shouldn't be committed or fetch via an initial clone
Besides there would be risks that a developer/contributor pushes some local changes without realizing it

Reply:
.env it's not a custom. It's actually the ".dist"

Reply:
Ah ok I didn't know so ok for adding a `.env` but for now I guess it should remain empty, we don't want to add the feature flags values because it would make the DB layer useless